examples/stable_diffusion_v2/ldm/data/dataset_controlnet.py

```diff
@@ -144,11 +144,9 @@ def tokenize(self, text):
 
         sot_token = self.tokenizer.encoder[SOT_TEXT]
         eot_token = self.tokenizer.encoder[EOT_TEXT]
-        pad_token = eot_token
         tokens = [sot_token] + self.tokenizer.encode(text) + [eot_token]
         # TODO: use pad token instead of zero
-        # result = np.zeros([CONTEXT_LEN])
-        result = np.array([pad_token] * CONTEXT_LEN) + eot_token
+        result = np.zeros([CONTEXT_LEN]) + eot_token
```

Comment:
Here is the same: describe the problem you're fixing and what you've done to fix it. A short sentence like "the pad token is added twice" would suffice.